### PR TITLE
[F] Add modal button icons and animations

### PR DESCRIPTION
**Pull request by alexgoff**
JIRA: https://jira.lsstcorp.org/browse/EPO-5804

## What this change does ##

Added icons to the text button and a default icon

## Notes for reviewers ##

Include an indication of how detailed a review you want on a 1-10 scale.
- 2

## Testing ##

Run Surveying the Solar System and proceed through to the "semi major axis" page. Observe the addition of the lightbulb and its animation that plays after 1s. 

## Checklist ##

If any of the following are true, please check them off
- [ ] This is a breaking change: changes in page data (`/src/data/pages/PAGE.json`) or scientific data (anything in `/static/`) and I will notify Product Marketing when it is in prod and user visible.
- [ ] This change impacts several investigations (e.g. the change affects reuseed styles, widgets, pages, QAs, utility methods, etc.)
- [x] This change is isolated to a specific page or Component (i.e. it's a one-off)
- [ ] I don't know what this change does

### CODE REVIEW

File: src/assets/stylesheets/lib/animate.css [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+  to {
+    -webkit-transform: translate3d(0, 0, 0);
+    transform: translate3d(0, 0, 0);
+  }
+}
+.animate__wobble {
+  -webkit-animation-name: wobble;
+  animation-name: wobble;
+}
+/* @import 'attention_seekers/jello.css'; */
```
Comment (alexgoff) [inline src/assets/stylesheets/lib/animate.css:200]: these commented imports are the result of the custom build. I can either manually delete, use the production build of animate.css, or simply leave as-is.

File: src/assets/stylesheets/styles.scss [hunk 2]
```diff
@@ -1,5 +1,4 @@
-@import 'lib/normalize';
-@import 'lib/md-globals';
+@import 'lib/index';
```
Comment (alexgoff) [inline src/assets/stylesheets/styles.scss:1]: moved the vendor files into a separate imports file

File: src/components/page/blocks/ReferenceBlock.jsx [hunk 3]
```diff
@@ -27,6 +27,8 @@ function ReferenceBlock({ block, blockShared, row }) {
   } = block;
 
   const { position } = options || 'left';
+  const defaultIcon = 'lightbulb';
+  const iconButton = button.icon && !button.text;
```
Comment (alexgoff) [inline src/components/page/blocks/ReferenceBlock.jsx:31]: `iconButton` is buttons which are _only_ icons, as opposed to a plain text button or a button with text and an icon.

File: src/data/pages/visible-light-filters.json [hunk 4]
```diff
@@ -89,7 +89,7 @@
   ],
   "reference": [
     {
-      "title": "Test Modal Title",
+      "title": "Wavelength Ranges of Visible Light",
```
Comment (alexgoff) [inline src/data/pages/visible-light-filters.json:92]: holdover from the mockup implementation of the modal.

File: src/data/pages/categorizing-new-discoveries.json [hunk 5]
```diff
@@ -29,6 +29,7 @@
     {
       "title": "Planetary Orbital Sizes",
       "button": {
+        "icon": true,
```
Comment (alexgoff) [inline src/data/pages/categorizing-new-discoveries.json:32]: use `icon: true` to indicate that an icon displays, then the presence or absence of text to decide if it's an icon button or a text button with an icon.

**Review by blnkt — APPROVED** [review]
Oh my goodness the wiggle-wobble on that lightbulb is just adorable.  I hope Jose and Ardis like.  Maaaaaybe Ardis will wanna see the whole button wobble instead...but I like this.  We'll see.

**Pull request merged**